src/routes/resources/resources.handlers.ts

```
@@ -13,14 +13,36 @@ import type {
+  type = type && isResourceType(type) ? type : undefined;
+  const cacheKey = isFiltered
+    ? `resources:${JSON.stringify({ type, category, q, tags })}`
+    : "resources";
+  const cacheData = await c.env.MY_KV.get(cacheKey);
   if (cacheData) {
     return c.json(JSON.parse(cacheData), HttpStatusCodes.OK);
   }
   const resources = await db.query.resources.findMany({
     orderBy: (resources, { desc }) => desc(resources.createdAt),
-    where: (resources, { eq }) => eq(resources.isPublished, true),
+    where: (resources, { and, eq, inArray }) =>
```

ESLint

[error] 'inArray' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)